internals/search_test.py

```diff
@@ -344,6 +344,11 @@ def test_process_query__negated_predefined(
 
   def test_process_query__single_field(self):
     """We can can run single-field queries."""
+    actual, tc = search.process_query('')
```

Comment:
We need some test cases with Features (and FeatureEntries) that have unlisted=true and deleted=true.

Reply:
Added!